app/models/notification.rb

```diff
@@ -2,8 +2,31 @@ class Notification < ActiveRecord::Base
   belongs_to :event
   # has_many :recipients
 
-  VALID_STATUSES = %w(pending active canceled complete expired)
-  validates :status, inclusion: { in: VALID_STATUSES }
+  enum status: [ :pending, :active, :canceled, :complete, :expired ]
 
+  before_create :validate_status
+  before_update :validate_status_change, if: :status_changed?
```

Comment:
https://github.com/ReadyResponder/ReadyResponder/issues/262 requires `pending status` to only be changed to `active` or `cancelled`. Unfortunately it suggested to use a complicated `state_machine`. (The gem is nearly 6000 total lines) Instead this can be managed with `enum` very easily. I just compare the previous values and control the change.